testutils/blockdata.go

```diff
@@ -0,0 +1,158 @@
+package testutils
```

Comment:
moved this here, as this repo is the only place that it is used. Happy to move it to celestia-core if we think that's a better plan.

Reply:
Can you pls point to the place we use those? Right now, in Share Service, we define our own analogs of such helpers.  If it is used in IPLD and absolutely necessary pls move those into `ipld` package, otherwise we don't need them.

Reply:
Oh, ok this is also used in `service/block`. Can you pls put each used testing helper in respective package where it is used? E.g `GenerateRandomBlockData` to be put into `testing.go` in `service/block`.